lib/vendor/noitidart-ostypes/cutils.jsm

```diff
@@ -0,0 +1,462 @@
+/*** CTypes Utilities ***
+ * Stuff that I find useful in my CTypes work under all Operating Systems
```

Comment:
Any benefits from copy-pasta-ing the [~12,400 lines of vendored code] into our repo versus adding https://github.com/Noitidart/ostypes as a [dev]dependency and copying the node_modules files into ./lib/vendor/*?